src/interfaces/swig/Library.i

```diff
@@ -399,100 +394,40 @@ namespace shogun
 %include <shogun/lib/StructuredDataTypes.h>
 %include <shogun/lib/StructuredData.h>
 %include <shogun/lib/DynamicObjectArray.h>
-%include <shogun/lib/WrappedObjectArray.h>
-%include <shogun/lib/IndexBlock.h>
-%include <shogun/lib/IndexBlockRelation.h>
-%include <shogun/lib/IndexBlockGroup.h>
-%include <shogun/lib/IndexBlockTree.h>
-%include <shogun/lib/Data.h>
-
-/* Computation framework */
-
-/* Computation Engine */
-%rename (IndependentComputationEngine) CIndependentComputationEngine;
-%rename (SerialComputationEngine) CSerialComputationEngine;
-
-%include <shogun/lib/computation/engine/IndependentComputationEngine.h>
-%include <shogun/lib/computation/engine/SerialComputationEngine.h>
-
-/* Independent compution-job */
-%include <shogun/lib/computation/job/IndependentJob.h>
-
-/* Independent computation-job results */
-%rename (JobResult) CJobResult;
-%include <shogun/lib/computation/jobresult/JobResult.h>
-%include <shogun/lib/computation/jobresult/ScalarResult.h>
+<<<<<<< HEAD
```

Comment:
you have some serious git issues in here ;)
Need to read on how to rebase and how all of git works really. Never commit merge conflicts into a PR

Reply:
Yeah I did make a lot mistakes sorry about that

Reply:
No worries at all. Next time, just make sure to have a look at the diff of your PR in github yourself before you press "send".